docs/en-us/user/references/metadata/introduction.md

```diff
@@ -0,0 +1,369 @@
+# Background
+There are close to [30 configurations](http://dubbo.apache.org/en-us/docs/user/references/xml/dubbo-service.html) in dubbo provider. Excluding registry center governance requirements, a large portion of configurations are used by the provider itself and do not need to be delivered to the consumer. This part of the data does not need to be written to the registry, but only needs to be persisted as key-value.
```

Comment:
`portion` --> `part`?

Where possible you should use a simple word over a complex word. Simple words are easier to read and let your readers focus on your ideas.

Reply:
Yes. Thank you for your good advice.